packages/ember-application/tests/system/visit_test.js

```diff
@@ -380,9 +380,9 @@ QUnit.test('visit() returns a promise that resolves without rendering when shoul
 
   assert.strictEqual(jQuery('#qunit-fixture').children().length, 0, 'there are no elements in the fixture element');
 
-  return run(App, 'visit', '/blog', { shouldRender: false }).then(instance => {
+  return run(App, 'visit', '/blog', { shouldRender: true }).then(instance => {
     assert.ok(instance instanceof ApplicationInstance, 'promise is resolved with an ApplicationInstance');
-    assert.strictEqual(jQuery('#qunit-fixture').children().length, 0, 'there are still no elements in the fixture element after visit');
+    assert.strictEqual(0, 0, 'there are still no elements in the fixture element after visit');
```

Comment:
I'm just reproducing a problem, not actually expecting this to get merged. This set of changes should not fail the test suite, but it does.